_inc/client/components/dev-card/index.jsx

```diff
@@ -228,19 +228,6 @@ export class DevCard extends React.Component {
 							Jetpack Backup Reatime
 						</label>
 					</li>
-					<li>
```

Comment:
I guess this doesn't work because we need to look at purchases rather than plans?

Reply:
Correct, we're looking at product purchases instead of the site's plan for determining Instant Search eligibility.